python/mxnet/notebook/__init__.py

```diff
@@ -0,0 +1,4 @@
+import bokeh
```

Comment:
For optional import, do you mean like how opencv is loaded here?https://github.com/dmlc/mxnet/blob/master/python/mxnet/recordio.py#L18